CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ IMPROVEMENTS:
  * core: Improved job deregistration error logging. [[GH-8745](https://github.com/hashicorp/nomad/issues/8745)]
  * api: Added support for cancellation contexts to HTTP API. [[GH-8836](https://github.com/hashicorp/nomad/issues/8836)]
  * driver/docker: upgrade pause container and detect architecture [[GH-8957](https://github.com/hashicorp/nomad/pull/8957)]
+ * client: consul-template updated to v0.25.0 [[GH-8988](https://github.com/hashicorp/nomad/pull/8988)]
```

Comment:
Can you call out the client config deprecated here as well?

There's currently a merge conflict on this CHANGELOG file. Also (and this probably seems silly-picky) we sort the changelog entries by `core:`, then other sections alphabetically, and then by line length. So if you could move this to just above the `driver/docker` entry that'd be great.

Reply:
Done.